cli/pkg/artifact_builder/artifact_builder.go

```diff
@@ -33,14 +33,18 @@ type ArtifactBuilder struct {
 	backend  *backend.Backend
 	config   *BuilderConfig
 	Profiler *profiler.Profiler
-	tags     []string
+	Tags     []string
```

Comment:
Do we need to export `Tags`? There's a `GetTags()` func. Two conflicting access methods.